openmetadata-ui/src/main/resources/ui/playwright/utils/lineage.ts

```diff
@@ -651,8 +654,10 @@ export const verifyExportLineagePNG = async (page: Page) => {
     }
   );
 
-  await page.getByTestId('export-type-select').click();
-  await page.locator('.ant-select-item[title="PNG"]').click();
+  if (!isPNGSelected) {
+    await page.getByTestId('export-type-select').click();
+    await page.locator('.ant-select-item[title="PNG"]').click();
+  }
```

Comment:
if isPNGSelected is false then you are clicking on PNG option?

Reply:
yes @ShaileshParmar11

Reply:
isn't that wrong?